src/main/java/io/vertx/ext/hawkular/VertxHawkularOptions.java

```diff
@@ -81,6 +83,16 @@
    */
   public static final boolean DEFAULT_METRICS_BRIDGE_ENABLED = false;
 
+  /**
+   * The default number of metric names to cache in order to avoid repeated tagging requests = 100.
+   */
+  public static final int DEFAULT_TAGGED_METRICS_CACHE_SIZE = 100;
```

Comment:
Question of fine-tuning of the default, I would image 100 is a bit too low: 100 map entries in memory is quite cheap whereas having additional HTTP calls to do unnecessary tagging is IMO more problematic. It's probably subject to debate.